website_membership_random_order/README.rst

```diff
@@ -0,0 +1,92 @@
+=======================================
+Online Members Directory - Random order
+=======================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fvertical--association-lightgray.png?logo=github
+    :target: https://github.com/OCA/vertical-association/tree/13.0/website_membership_random_order
+    :alt: OCA/vertical-association
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/vertical-association-13-0/vertical-association-13-0-website_membership_random_order
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/208/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This modules extends the online members directory for displaying the results
+in a random order, and displays a disclaimer about it at the top of the list.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Known issues / Roadmap
+======================
+
+- All the members will be displayed at once, without pager, as this is the only
+  way to get a whole randomized list. On contrary, paging the content, on each
```

Comment:
Maybe you could use an already randomized vale from the user session (i.e.: cookie id) and use it to get a reliable order. A snippet from: https://stackoverflow.com/a/19307329

```python
>>> import random
>>> x = [1, 2, 3, 4, 5, 6]
>>> random.Random(4).shuffle(x)
>>> x
[4, 6, 5, 1, 3, 2]
>>> x = [1, 2, 3, 4, 5, 6]
>>> random.Random(4).shuffle(x)
>>> x
[4, 6, 5, 1, 3, 2]
```

Reply:
But it's not the intended goal, as for that session, the order will be always the same, and thus the top results won't change. We prefer to show everything and each time you access, see different top results.

Reply:
Ok, ok. I guess it could be achieved with a more complex implementation, but for the present purposes it will suffice